### PR TITLE
Enable golangci in presubmit

### PR DESCRIPTION
**Pull request by yfei1**
https://github.com/golangci/golangci-lint/issues/500 is fixed.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -40,6 +40,5 @@ require (
 	github.com/stretchr/testify v1.3.0
```
Comment (Laremere) [inline go.mod]: this seems like an unrelated change?

File: internal/app/backend/backend_service.go [hunk 2]
```diff
@@ -42,8 +42,7 @@ type backendService struct {
 }
 
 type mmfClients struct {
-	addrMap    sync.Map
```
Comment (Laremere) [inline internal/app/backend/backend_service.go]: this seems like an unrelated change?

File: go.mod (hunk 1 above)
Comment (yfei1) [reply go.mod]: This is done by the `go mod tidy` command in `make presubmit` automatically.

File: internal/app/backend/backend_service.go (hunk 2 above)
Comment (Laremere) [reply internal/app/backend/backend_service.go]: Per discussion: golangci complains that this is unused.  Therefor removing.
